public_suffix_list.dat

```diff
@@ -258,7 +258,7 @@ tas.gov.au
 vic.gov.au
 wa.gov.au
 // 4LDs
-education.tas.edu.au
+// education.tas.edu.au - Removed at the request of the Department of Education Tasmania
```

Comment:
- Passes test;
- No conflict with base
- Rationale for change valid

DNS Validation was not performed, but a **reliable alternative means** was used to demonstrate **authority and connection to validate the request comes from an administrative manager** of the requested resource using WHOIS on a registered string.

The oversight of the .AU namespace happens via AUDA.AU, at the URL: auda.org.au.  Doing a lookup in the whois returned a result which hard coded the PR number in, proving the connection in a similar manner to the typical DNS validation model. 

```
Domain Name: PSL.EDUCATION.TAS.EDU.AU
Registry Domain ID: D407400000093511041-AU
Registrar WHOIS Server: whois.auda.org.au
Registrar URL: https://www.domainname.edu.au
Last Modified: 2020-06-03T01:06:50Z
Registrar Name: EDUCATION SERVICES AUSTRALIA LIMITED
Registrar Abuse Contact Email: registrar@esa.edu.au
Registrar Abuse Contact Phone: +61.399109829
Reseller Name:
Status: inactive https://afilias.com.au/get-au/whois-status-codes#inactive
Status: serverRenewProhibited https://afilias.com.au/get-au/whois-status-codes#serverRenewProhibited
Status: addPeriod https://afilias.com.au/get-au/whois-status-codes#addPeriod
Registrant Contact ID: PSL-977
Registrant Contact Name: kris-esa - https://github.com/publicsuffix/list/pull/977
Registrant Contact Email: kris.cutmore@esa.edu.au
Tech Contact ID: PSL-977
Tech Contact Name: kris-esa - https://github.com/publicsuffix/list/pull/977
Tech Contact Email: kris.cutmore@esa.edu.au
Name Server:
DNSSEC: unsigned
Registrant: EDUCATION SERVICES AUSTRALIA LIMITED
Registrant ID: ABN 18007342421
Eligibility Type: Government Body
Eligibility Name: https://github.com/publicsuffix/list/pull/977
```

The entry in question is a specialized TLD namespace within the .AU ccTLD.  By proving connection in the manner used, I believe that the submitter has proven and verified authority in an acceptable manner and this should proceed.